### PR TITLE
Allow disabling `clean-mergeability-box` + fix some CSS leak

### PR DESCRIPTION
**Pull request by mskelton**
This makes `clean-mergeability-box` a feature and wraps the CSS for `extend-diff-expander` and `show-names` behind the associated feature classes.

## Screenshot

image

### CODE REVIEW

File: source/features/show-names.tsx [hunk 1]
```diff
@@ -78,6 +78,7 @@ function init(signal: AbortSignal): void {
 	observe(usernameLinksSelector, batchUpdateLinks, {signal});
 }
 
+void features.addCssFeature(import.meta.url);
```
Comment (fregante) [inline source/features/show-names.tsx:81]: This should only be used for CSS-only features. If there's already an `init` function we can add the class as `document.body.classList.add(something)`

File: source/features/clean-mergeability-box.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import './clean-mergeability-box.css';
+
+import features from '../feature-manager';
+
+void features.addCssFeature(import.meta.url);
```
Comment (fregante) [inline source/features/clean-mergeability-box.tsx:5]: This needs a pageDetect filter: isPRConversation

File: readme.md [hunk 3]
```diff
@@ -395,6 +395,7 @@ Thanks for contributing! 🦋🙌
 - [](# "command-palette-navigation-shortcuts") Adds keyboard shortcuts to select items in command palette using <kbd>ctrl</kbd> <kbd>n</kbd> and <kbd>ctrl</kbd> <kbd>p</kbd> (macOS only).
 - [](# "submission-via-ctrl-enter-everywhere") Enables submission via <kbd>ctrl</kbd> <kbd>enter</kbd> on every page possible.
 - [](# "clean-repo-header") [Reduces repository name wrapping by hiding the fork and watch count (they're still on the repository’s home page)](https://user-images.githubusercontent.com/1402241/218252904-6a31a933-41a7-452e-b841-9484e67429a8.png)
+- [](# "clean-mergeability-box") [Removes extra information in the PR mergeability box.](https://user-images.githubusercontent.com/25914066/224514342-ba144942-95c4-46ec-a4cd-0836ea6668f5.png)
```
Comment (fregante) [inline readme.md:398]: The screenshot doesn't show what's being removed, can you do a before/after like we do for some other features? Please reuse the same exact "before/after" text and arrows (you can literally copy-paste it from some other screenshot)

**Review by fregante — COMMENTED** [review]
Thanks for the PR!

File: readme.md (hunk 3 above)
Comment (mskelton) [reply readme.md]: Updated!

File: source/features/show-names.tsx (hunk 1 above)
Comment (fregante) [reply source/features/show-names.tsx]: This applies to all

Comment (mskelton) [reply source/features/show-names.tsx]: Oops, missed the other one, thanks!

File: source/features/github-bugs.css [hunk 4]
```diff
@@ -53,6 +53,11 @@
 	vertical-align: baseline !important;
 }
 
+/* Fix spacing of repo header button icons #5620 */
+.pagehead-actions :is(.btn, summary) .octicon:not(.octicon-triangle-down) {
+	margin-right: 4px !important;
```
Comment (fregante) [inline source/features/github-bugs.css:58]: Thank you!

**Review by fregante — APPROVED** [review]

**Pull request merged**